File: solutions/problem_0013/java/EmpresaOnibusProject/model/Aposentado.java
package ExerciciosFixacao.Fixacao4.EmpresaOnibusProject.model;

public class Aposentado extends Passageiro{
    private int anoNascimento;
    private String rg;

    public Aposentado(String nome, int anoNascimento, String rg){
        super(nome);
        if(anoNascimento > 1900){
            this.anoNascimento = anoNascimento;
        }else{
            this.anoNascimento = 2000;
        }
        this.rg = rg;
    }

    public int getAnoNascimento() {
        return anoNascimento;
    }

    public void setAnoNascimento(int anoNascimento) {
        this.anoNascimento = anoNascimento;
    }

    public String getRg() {
        return rg;
    }

    public void setRg(String rg) {
        this.rg = rg;
    }

    @Override
    public double getValorPagamento(double tarifa) {
        this.tarifa = tarifa;
        return 0;
    }

    @Override
    public String toString() {
        return super.toString() + "\nNascimento: " + anoNascimento + "\nRG: " + rg;
    }

    
}

File: solutions/problem_0013/kotlin/EmpresaOnibusProject/src/model/Aposentado.kt
package model

class Aposentado(nome: String, inteira: Double, val rg: String, var anoNascimento: Int) : Passageiro(nome, inteira) {
    init {
        if (anoNascimento < 1900)
            anoNascimento = 2000;
    }

    override fun getValorPagamento(): Double {
        return 0.0
    }

    override fun toString(): String {
        return "${super.toString()}\nRG: $rg \nNascido(a) em: $anoNascimento"
    }
}